themes/dark-aliceblue-mineral.css

```diff
@@ -0,0 +1,95 @@
+/* dark-aliceblue-mineral theme v1.0.0 by Axorax - https://github.com/Axorax/mit-license-themes-axorax
+ * MIT License - https://axorax.mit-license.org/
+ * Preview - https://axorax.github.io/mit-license-themes-axorax/preview?theme=dark-aliceblue-mineral
+ */
+
+ @import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
+
+ :root {
+     color-scheme: dark;
+ }
+ 
+ body {
+   margin: 0;
+   font-family: 'Source Sans Pro', sans-serif;
+   line-height: 1.5;
+   background: #121212;
+   color: #f5f5f5;
+ }
+ 
+ a {
+     text-decoration: none;
+     color: #f0f8ff;
+ }
+ 
+ a:hover, a:focus {
+     text-decoration: underline;
+ }
+ 
+ article, footer {
+     width: 51.5rem;
+     max-width: calc(100% - 6rem);
+     margin: 1.2em auto 0 auto;
+     border-radius: 5px;
+     padding: 2rem;
+     background: #171717;
+     border: 1.3px solid #f0f8ff;
+     font-size: 16px;
+ }
+ 
+ article p:nth-last-child(2) {
+     border-left: 3px solid #f0f8ff;
+     padding: .8rem 1.2rem;
+     border-radius: 7px;
+     background: #f0f8ff59;
+     color: #f0f8ff;
+ }
+ 
+ footer {
+     width: 54rem;
+     max-width: calc(100% - 3.6rem);
+     text-align: center;
+     padding: 0 .8rem!important;
```

Comment:
Consider inserting a space between the end of the unit and `!important`

Reply:
Done! Inserted space before !important in all of the css files.